src/generic/grid.cpp

```diff
@@ -10337,4 +10332,44 @@ wxGridCellEditor* wxGridTypeRegistry::GetEditor(int index)
     return editor;
 }
 
+wxRect wxGetGridCheckBoxRect(const wxSize& checkBoxSize,
+                             const wxRect& cellRect,
+                             int hAlign, int WXUNUSED(vAlign))
+{
+    // TODO: support vAlign
+
+    wxRect checkBoxRect;
+    checkBoxRect.SetY(cellRect.y + cellRect.height / 2 - checkBoxSize.y / 2);
+
+    wxCoord minSize = wxMin(cellRect.width, cellRect.height);
+    if ( checkBoxRect.GetWidth() >= minSize || checkBoxRect.GetHeight() >= minSize )
```

Comment:
This was supposed to be
```suggestion
    if ( cellRect.GetWidth() >= minSize || cellRect.GetHeight() >= minSize )
```
right? Otherwise this condition is just always false as `checkBoxRect` size is 0 here.

Reply:
Yes, it was the error.